packages.ini

```diff
@@ -372,10 +372,13 @@ python_versions = <3.11
 python_versions = <3.11
 [grpcio==1.51.1]
 python_versions = <3.11
+[grpcio==1.56.0]
+python_versions = <3.11
```

Comment:
this isn't right -- grpcio 1.56 supports 3.11

Reply:
I'll get that fixed. 😓